Cabal/Distribution/Types/GenericPackageDescription.hs

```diff
@@ -61,6 +65,9 @@ data Flag = MkFlag
 
 instance Binary Flag
 
+emptyFlag :: FlagName -> Flag
```

Comment:
Well, when I look at the code, I wonder, "Why is it True and then False?" And then I look at `Flag` and I see that the first is `flagDefault` and `flagManual`. At this point I can infer that the default settings for a flag are for it to be (1) turned on by default and (2) not manual (whatever that means...) The comment should explain something like that?